index.html

```diff
@@ -4023,15 +4013,9 @@ <h3>Reserved Extension Points</h3>
             <tr>
               <td>`renderMethod`</td>
               <td>
-A property used for specifying one or more methods to render a credential into a visual,
-auditory, or haptic format. The associated vocabulary URL MUST be
+A property used for specifying one or more methods to render a credential into a
+visual, auditory, or haptic format. The associated vocabulary URL MUST be
```

Comment:
I don't think an aromatic, flavor, or other sensory format is likely, but it seems more future proof to allow for such.
```suggestion
visual, auditory, haptic, or other format. The associated vocabulary URL MUST be
```

Reply:
Verifiable smells.